modules/op-supported-keys-tekton-chains-configuration.adoc

```diff
@@ -9,48 +9,279 @@
+
+.Chains configuration: Supported keys for Grafeas storage
+[options="header"]
+|===
+
+| Key | Description | Supported values | Default value
+
+| `storage.grafeas.projectid`
+| The {OCP} project in which the grafeas server for storing occurrences is located
+|
+|
+
+| `storage.grafeas.noteid`
+| Optional: the prefix to use for the name of all created notes.
+| A string without spaces.
+|
+
+| `storage.grafeas.notehint`
+| Optional: the https://github.com/grafeas/grafeas/blob/cd23d4dc1bef740d6d6d90d5007db5c9a2431c41/proto/v1/attestation.proto#L49[human_readable_name] field for the Grafeas `ATTESTATION` note.
+|
+|`This attestation note was generated by Tekton Chains`
+|===
+
+Optionally, you can enable additional uploads of binary transparency attestations.
+
+.Chains configuration: Supported keys for transparency attestation storage
+[options="header"]
+|===
+
+| Key | Description | Supported values | Default value
+
+| `transparency.enabled`
+| Enable or disable automatic binary transparency uploads.
+| `true`, `false`, `manual`
+| `false`
+
+| `transparency.url`
+| The URL for uploading binary transparency attestations, if enabled.
+|
+| `+https://rekor.sigstore.dev+`
+|===
+
+NOTE: If you set `transparency.enabled` to `manual`, only task runs and pipeline runs with the following annotation are uploaded to the transparency log:
+
+[source,yaml]
+----
+chains.tekton.dev/transparency-upload: "true"
+----
+
+If you configure the `x509` signature backend, you can optionally enable keyless signing with Fulcio.
+
+.Chains configuration: Supported keys for x509 keyless signing with Fulcio
+[options="header"]
+|===
+
+| Key | Description | Supported values | Default value
+
+| `signers.x509.fulcio.enabled`
+| Enable or disable requesting automatic certificates from Fulcio.
+| `true`, `false`
+| `false`
+
+| `signers.x509.fulcio.address`
+| The Fulcio address for requesting certificates, if enabled
+|
+| `+https://v1.fulcio.sigstore.dev+`
+
+| `signers.x509.fulcio.issuer`
+| The expected OIDC issuer.
+|
+| `+https://oauth2.sigstore.dev/auth+`
+
+| `signers.x509.fulcio.provider`
+| The provider from which to request the ID Token
+| `google`, `spiffe`, `github`, `filesystem`
+| {pipelines-title} attempts to use every provider
+
+| `signers.x509.identity.token.file`
+| Path to the file containing the ID Token.
+|
+|
+
+| `signers.x509.tuf.mirror.url`
+| The URL for the TUF server. `$TUF_URL/root.json` must be present.
+|
+| `+https://sigstore-tuf-root.storage.googleapis.com+`
+|===
+
+If you configure the `kms` signature backend, set the KMS configuration, including OIDC and Spire, as necessary.
+
+.Chains configuration: Supported keys for KMS signing
+[options="header"]
+|===
+
+| Key | Description | Supported values | Default value
```

Comment:
Does it make sense to have two completely empty columns here?

Reply:
So upstream the two columns are empty, not sure if we should remove or keep this two empty columns

Reply:
This is not about upstream - this is about consistency with all the other tables under this same heading, where the same columns are not empty. I would prefer to have more information about these keys but I don't have any.